tests/unit/amo/components/TestAddonReviewListItem.js

```diff
@@ -188,6 +189,58 @@ describe(__filename, () => {
     );
   });
 
+  it('renders a delete link for a user review', () => {
+    const review = signInAndDispatchSavedReview();
+    const root = render({ review });
+
+    const deleteLink = renderControls(root).find('.AddonReviewListItem-delete');
+    expect(deleteLink).toHaveLength(1);
+    expect(deleteLink.children()).toHaveText('Delete my review');
+    expect(deleteLink).toHaveProp(
+      'message',
+      'Do you really want to delete this review?',
+    );
+  });
+
+  it('does not render delete link when review belongs to another user', () => {
+    const review = signInAndDispatchSavedReview({
+      siteUserId: 123,
+      reviewUserId: 987,
+    });
+    const root = render({ review });
+
+    expect(
+      renderControls(root).find('.AddonReviewListItem-delete'),
+    ).toHaveLength(0);
+  });
+
+  it('dispatches deleteReview when a user clicks the delete link', () => {
+    const review = signInAndDispatchSavedReview();
+    const dispatchSpy = sinon.spy(store, 'dispatch');
+    const preventDefaultSpy = sinon.spy();
+    const root = render({ review });
+    const { errorHandler } = root.instance().props;
+
+    const deleteButton = renderControls(root).find(
+      '.AddonReviewListItem-delete',
+    );
+    const clickEvent = createFakeEvent();
+    deleteButton.simulate('click', clickEvent);
```

Comment:
This is clicking `<ConfirmButton>`, right? I don't think that's necessary and it's also not a good idea because it reaches into the implementation of `ConfirmButton`. You can trust that `ConfirmButton` works as advertised. All you need to do is make sure its props are set correctly.

Reply:
Yeah, that was just me forgetting to remove that line when I changed it to just call the `onConfirm` prop.